composer.json

```diff
@@ -24,8 +24,8 @@
   ],
   "require": {
     "php": "^7.2|^8.0",
-    "doctrine/orm": "~2.6.0|~2.7.0|~2.8.0|~2.9.5",
-    "doctrine/inflector": "^1.1",
+    "doctrine/orm": "^2.6",
+    "doctrine/inflector": "^1.4|^2.0",
     "doctrine/persistence": "^1.3.5|^2.0"
```

Comment:
`Doctrine\DBAL\Types\Types` was introduced in doctrine/dbal 2.10. Using composer update --prefer-lowest (on PHP 7.4) installs dbal 2.6 Think we should require dbal ^2.10 now.

https://github.com/doctrine/dbal/releases/tag/v2.10.0

Reply:
Ok. I'll add 2.10 and 3.3 as dependencies then